diff --git a/node_modules/expo-router/build/global-state/router-store.js b/node_modules/expo-router/build/global-state/router-store.js
index 05ad58f..2a6c975 100644
--- a/node_modules/expo-router/build/global-state/router-store.js
+++ b/node_modules/expo-router/build/global-state/router-store.js
@@ -209,7 +209,7 @@ function useStore(context, linkingConfigOptions, serverUrl) {
                 splashScreenAnimationFrame = undefined;
             }
         };
-    });
+    }, []);
     return exports.store;
 }
 const routeInfoSubscribers = new Set();
